testsuite/gna/issue2005/testsuite.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+
+. ../../testenv.sh
+
+set -x
+
+if c_compiler_is_available; then
+	lib=$("$GHDL" --libghdl-name)
+	lib=${lib#lib} # strip "lib" prefix
+	lib=${lib%.so} # strip .so suffix
+
+	$CC -I"$("$GHDL" --libghdl-include-dir)" -L"$(dirname "$("$GHDL" --libghdl-library-path)")" test.c -l$lib
```

Comment:
Could be much simpler: `$CC "-I$("$GHDL" --libghdl-include-dir)" test.c "$("$GHDL" --libghdl-library-path)"`

Reply:
There's a reason I do it the long way around. This way I'm testing both --libghdl-library-path and --libghdl-library-name.